streamz/core.py

```diff
@@ -675,6 +690,20 @@ def update(self, x, who=None):
         return self.emit(x)
 
 
+class pluck(Stream):
+    def __init__(self, child, pick):
+        self.pick = pick
+        super(pluck, self).__init__(child)
+
+    def update(self, x, who=None):
+        L = []
+        # Note will raise IndexError if x is too small
+        for elem in self.pick:
+            L.append(x[elem])
+
+        return self.emit(L)
```

Comment:
I think that consistency with toolz would be something like the following:

```python
def update(self, x, who=none):
    if isinstance(self.pick, list):
        return self.emit(tuple([x[ind] for ind in self.pick]))
    else:
        return self.emit(x[self.pick])
```

Reply:
ok will add it (ignore the commit i just sent, was done before reading this)